virtme/commands/run.py

```diff
@@ -1282,25 +1282,27 @@ def do_it() -> int:
     if args.graphics is None and not args.script_sh and not args.script_exec:
         qemuargs.extend(["-echr", "1"])
 
-        if args.verbose:
-            # Check if we have permission to access the current stderr.
-            if not can_access_file("/proc/self/fd/2"):
-                print(
-                    "ERROR: not a valid pts, try to run vng inside tmux or screen",
-                    file=sys.stderr,
-                )
-                sys.exit(1)
-
-            # Redirect kernel messages to stderr, creating a separate console
+        # Redirect kernel errors to stderr, creating a separate console.
+        #
+        # If we don't have access to stderr via procfs (for example when
+        # running inside a container), print a warning and implicitly
+        # supress the kernel errors redirection.
+        if can_access_file("/proc/self/fd/2"):
             qemuargs.extend(["-chardev", "file,path=/proc/self/fd/2,id=dmesg"])
             qemuargs.extend(["-device", arch.virtio_dev_type("serial")])
             qemuargs.extend(["-device", "virtconsole,chardev=dmesg"])
             kernelargs.extend(["console=hvc0"])
+        else:
+            print(
+                "WARNING: not a valid pts, try to run vng inside tmux or screen",
+                file=sys.stderr,
```

Comment:
just to be sure: can we still write to stderr here if we cannot access `/proc/self/fd/2`?

Also, compared to before, this warning will always be printed in such environment, when stderr is not accessible. I guess it will not be an issue, right?